plugins/source/pagerduty/codegen/recipes/base.go

```diff
@@ -250,7 +247,8 @@ func SetParentChildRelationships(resources []*Resource) error {
 			name := csr.ToSnake(strings.TrimSuffix(ch, "()"))
 			v, ok := m[name]
 			if !ok {
-				return errors.New("child not found for " + r.SubService + " : " + name)
+				fmt.Println("WARNING: child not found for " + r.SubService + " : " + name)
```

Comment:
Is this expected to always raise a warning now? Maybe we can add an exception for the manually created table so that it doesn't confuse new contributors?